src/Avalonia.ReactiveUI/TransitioningContentControl.cs

```diff
@@ -62,11 +66,14 @@ public class TransitioningContentControl : ContentControl, IStyleable
         /// <param name="content">New content to set.</param>
         private async void UpdateContentWithTransition(object? content)
         {
+            _lastTransitionCts?.Cancel();
```

Comment:
Dispose()?

Reply:
No need. Token source was already cancelled and it didn't even kept a timer to dispose.